packages/parse5-htmlparser2-tree-adapter/package.json

```diff
@@ -26,7 +26,7 @@
         "parse5": "^7.0.0"
     },
     "scripts": {
-        "build:cjs": "tsc --module CommonJS --target ES6 --outDir dist/cjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json"
+        "build:cjs": "tsc --moduleResolution node10 --module CommonJS --target ES6 --outDir dist/cjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json"
```

Comment:
typescript started to complain without this, since you _must_ have `module: "node16"` if you have module resolution set to it (which we do)

I guess it got updated in the lockfile while updating tseslint etc and introduced the new behaviour